docs/contributing/frontend.md

```diff
@@ -237,13 +271,16 @@ another page, you should probably consider using the **E2E** approach.
 
 ### Visual testing
 
-Test components without user interaction like testing if a page view is rendered
-correctly depending on some parameters, if the button is showing a spinner if
-the `loading` props are passing, etc. This should always be your first option
-since it is way easier to maintain. For this, we use
+Test components without user interaction like testing if a page/component is
+rendered correctly depending on some parameters, if the button is showing a
+spinner, if the `loading` props are passing, etc. This should always be your
+first option since it is way easier to maintain. For this, we use
```

Comment:
> This should always be your first option since it is way easier to maintain.

I disagree with this, tbh. I think we're running into difficulties when we use visual testing as our first line of defense. Maybe something we can chat about more during the FE variety call!